Owncloud iOs Client/Network/Authentication/DetectAuthenticationMethod.swift

```diff
@@ -37,7 +37,7 @@ import Foundation
         request.setValue(UtilsUrls.getUserAgent(), forHTTPHeaderField: "User-Agent")
         request.setValue("application/xml", forHTTPHeaderField: "Content-Type")
         let body =  "<?xml version=\"1.0\" encoding=\"UTF-8\"?><D:propfind xmlns:D=\"DAV:\"><D:prop><D:resourcetype/><D:getlastmodified/><size xmlns=\"http://owncloud.org/ns\"/><D:creationdate/><id xmlns=\"http://owncloud.org/ns\"/><D:getcontentlength/><D:displayname/><D:quota-available-bytes/><D:getetag/><permissions xmlns=\"http://owncloud.org/ns\"/><D:quota-used-bytes/><D:getcontenttype/></D:prop></D:propfind>"
-        request.httpBody = body.data(using: String.Encoding.utf8)
+        //request.httpBody = body.data(using: String.Encoding.utf8)
```

Comment:
Why is this line commented?

Reply:
Was not very clear if we should remove the http body or not. Looks like we have to remove it so I will remove that line 👍